notebooks/Numba_054_DPPY_Release_Demo.ipynb

```diff
@@ -0,0 +1,341 @@
+    "\n",
+    "* `numba_dppy.get_global_id`: Gets the global ID of the item\n",
+    "* `numba_dppy.get_local_id`: Gets the local ID of the item\n",
+    "* `numba_dppy.get_local_size`: Gets the local work group size of the device\n",
+    "* `numba_dppy.get_group_id` : Gets the group ID of the item\n",
+    "* `numba_dppy.get_num_groups`: Gets the number of gropus in a worksgroup  \n",
+    "\n",
+    "Refer https://intelpython.github.io/numba-dppy/latest/user_guides/kernel_programming_guide/index.html for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "059b28de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dppy.kernel\n",
+    "def sum(a, b, c):\n",
+    "    i = dppy.get_global_id(0)\n",
+    "    c[i] = a[i] + b[i]\n",
+    "\n",
+    "a = np.array(np.random.random(20), dtype=np.float32)\n",
+    "b = np.array(np.random.random(20), dtype=np.float32)\n",
+    "c = np.ones_like(a)\n",
+    "\n",
+    "device = dpctl.select_default_device()\n",
+    "\n",
+    "with dpctl.device_context(device):\n",
+    "    sum[20, dppy.DEFAULT_LOCAL_SIZE](a, b, c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f050163e-d3b6-4bd8-888c-010d9f7b8e2c",
+   "metadata": {},
+   "source": [
+    "## Numba-DPPY Atomics\n",
+    "\n",
+    "Numba-dppy supports several atomic operations supported by DPC++.  \n",
+    "\n",
+    "`class numba_dppy.ocl.stubs.atomic`  \n",
+    "atomic namespace\n",
+    "\n",
+    "* `add(ary, idx, val)` Perform atomic `ary[idx] += val`. Returns the old value at the index location as if it is loaded atomically.\n",
+    "* `sub(ary, idx, val)` Perform atomic `ary[idx] -= val`. Returns the old value at the index location as if it is loaded atomically."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6c7694d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def main():\n",
+    "    \"\"\"\n",
+    "    The example demonstrates the use of numba_dppy's ``atomic_add`` intrinsic\n",
+    "    function on a SYCL GPU device. The ``dpctl.select_gpu_device`` is\n",
+    "    equivalent to ``sycl::gpu_selector`` and returns a sycl::device of type GPU.\n",
+    "\n",
+    "    If we want to generate native floating point atomics for spported\n",
+    "    SYCL devices we need to set two environment variables:\n",
+    "    NUMBA_DPPY_ACTIVATE_ATOMCIS_FP_NATIVE=1\n",
+    "    NUMBA_DPPY_LLVM_SPIRV_ROOT=/path/to/dpcpp/provided/llvm_spirv\n",
+    "\n",
+    "    To run this example:\n",
+    "    NUMBA_DPPY_ACTIVATE_ATOMCIS_FP_NATIVE=1 \n",
+    "    NUMBA_DPPY_LLVM_SPIRV_ROOT=/path/to/dpcpp/provided/llvm_spirv python atomic_op.py\n",
+    "\n",
+    "    Without these two environment variables Numba_dppy will use other\n",
+    "    implementation for floating point atomics.\n",
+    "    \"\"\"\n",
```

Comment:
The docstring seems to imply that the atomic only works on GPU devices, but it appears to be working on the CPU as well. Also, the description needs to be adapted for the notebook; e.g. the file `atomic_op.py` do not exist.